packages/knack-cli/cmds/runProducer.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs-extra');
+const KnackProducerClient = require('knack-producer-client');
+
+const tryParse = str => {
+	try {
+		return JSON.parse(str);
+	} catch (_) {
+		return str;
+	}
+};
+
+const resolveContent = async filePath => {
+	if (filePath) {
+		return tryParse(await fs.readFile(filePath));
+	}
+};
+
+const main = async (keyFilePath, valuefilePath, topic, count) => {
+	const key = await resolveContent(keyFilePath);
+	const value = await resolveContent(valuefilePath);
+
+	// TODO: make producer and topic config configurable
+	await KnackProducerClient.connectInstance();
+
+	const producer = KnackProducerClient.instance();
+
+	const tasks = [];
+
+	for (let i = 1; i <= count; i++) {
+		tasks.push(producer.publish({
+			topic,
+			key,
+			value
+		}));
+	}
+
+	await Promise.all(tasks);
+
+	await KnackProducerClient.disconnectInstances();
+
+	return {
+		message: `${tasks.length} messages sent to ${topic}`
```

Comment:
Suggestion so that the text of the message still makes sense when the count is 1.
``` 
message: `${tasks.length} message(s) sent to ${topic}`
```

Reply:
suggested update made